microprofile/tests/tck/tck-inject/tck-inject-test/pom.xml

```diff
@@ -41,8 +41,8 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>io.helidon.microprofile.bundles</groupId>
-            <artifactId>helidon-microprofile</artifactId>
+            <groupId>io.helidon.microprofile.cdi</groupId>
```

Comment:
If this module depends on the arquillian, then this should not be needed, otherwise this is OK

Reply:
It does not depend on Arquillian. What dependency should we add here?. Tests are located under microprofile, so `io.helidon.microprofile.cdi` seems correct to me.